### DISCUSSION TITLE
About only charging experiment

### DISCUSSION THREAD

**Discussion by Jadhav-Sourabh** [pybamm-team/PyBaMM#2135] [General]
![image](https://user-images.githubusercontent.com/87997086/176593274-0af8a38d-9c82-4360-a4cd-6a517cda59f1.png)
while adding the only charging experiment, I am getting this error, and in plot following problem is getting, voltage is starting from upper cut-off voltage, please someone provide me a proper solution for this problem
![image](https://user-images.githubusercontent.com/87997086/176593457-0dfa62d2-7ceb-48f1-93c6-546e0245e16d.png)

**Comment by brosaplanella** [comment]
I think this might be because you already start with a fully charge battery (which parameters are you using?), so when applying a charging 1C you are already above the maximum voltage and that's why it doesn't stop. Do you get a better behaviour if you first discharge for a while and then you charge?
